packages/st2mistral/Makefile

```diff
@@ -8,7 +8,7 @@ PBR_GITURL := git+https://github.com/openstack-dev/pbr@d19459daa8616dd18fde016c2
 
 # We use special additional requirements in our mistral bundle!
 define INJECT_DEPS
-git+https://github.com/StackStorm/st2mistral.git@$(GITREV)#egg=st2mistral
+git+https://github.com/StackStorm/st2mistral.git@st2-$(MISTRAL_VERSION)#egg=st2mistral
```

Comment:
So yeah, as @m4dcoder noted, looks it doesn't fix anything.
`GITREV` value should be `st2-2.2.1` from the https://github.com/StackStorm/st2-packages/blob/v2.2/rake/build/environment.rb#L85

Eg. correct value is already provided.